cmake/OmimTesting.cmake

```diff
@@ -0,0 +1,87 @@
+include(OmimConfig)
+
+# TestServer fixture configuration
+add_test(
+  NAME OmimStartTestServer
+  COMMAND start_server.py
+  WORKING_DIRECTORY ${OMIM_ROOT}/tools/python/test_server
+)
+add_test(
+  NAME OmimStopTestServer
+  COMMAND stop_server.py
+  WORKING_DIRECTORY ${OMIM_ROOT}/tools/python/test_server
+)
+set_tests_properties(OmimStartTestServer PROPERTIES FIXTURES_SETUP TestServer)
```

Comment:
Will the python test server start for _any_ test, even when it's not needed?

Reply:
No. It's a fixture configuration.
Server will start only for tests marked with `REQUIRE_SERVER`.
This fixture will be added for a test as a dependency:
```cmake
if (require_server)
    set_tests_properties(${TEST_NAME} PROPERTIES FIXTURES_REQUIRED TestServer)
endif()
```

Reply:
It's also woth to notice that `OmimStartTestServer` and `OmimStopTestServer` will be called only once.